edu.cuny.hunter.log.evalution/src/edu/cuny/hunter/log/evaluation/handlers/EvaluationHandler.java

```diff
@@ -59,13 +59,15 @@
 	private static final String EVALUATION_PROPERTIES_FILE_NAME = "eval.properties";
 	private static final String NOT_LOWER_LOG_LEVEL_IF_STATEMENT_KEY = "edu.cuny.hunter.log.evaluation.notLowerLogLevelInIfStatement";
 	private static final String NOT_LOWER_LOG_LEVEL_CATCH_BLOCK_KEY = "edu.cuny.hunter.log.evaluation.notLowerLogLevelInCatchBlock";
+	private static final String NOT_LOWER_LOG_LEVEL_KEYWORDS_KEY = "edu.cuny.hunter.log.evaluation.notLowerLogLevelWithKeywords";
```

Comment:
Is this documented in the wiki?

Reply:
Yes. https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/wiki/Running-the-Evaluator